web3j/src/main/resources/templates/report.html

```diff
@@ -44,6 +44,20 @@
   <div>
     <br>
     <br>
+    <h4>Token Balance</h4>
+    <table class="table table-dark" id="accDetails">
+      <tbody>
+      <tr>
+        <th>Sender</th>
+        <td th:text="${progress.senderBalance}"></td>
+      </tr>
+      <tr th:each="recipient,recipientStat : ${progress.recipientBalance}">
+        <th th:text="'Recipient ' + ${recipientStat.index+1}"></th>
```

Comment:
As we discussed today morning, let's make the account numbers available in the UI. Since the number are too big, maybe just show first few characters.

```
Sender (CB036..)
Recipient (45412..)
Recipient (1FF48..)

Reply:
I was thinking recipient 1,2... are better because even if a single digit is changed in recipient address in last few digits that can be a bit confusing for users. For example. R1 = 1FF48...0 and R2= 1FF48..1 these are 2 different recipients but their initial addresses are the same. Recipient 1,2,3 ... looks more clean and simple I guess.

Reply:
Check with @ramkri123 if we want recipient account address to be part of tooltip or something.

Reply:
Okay sure